modules/couchbase/README.md

```diff
@@ -0,0 +1,58 @@
+<!--
+title: "Couchbase monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/go.d.plugin/edit/master/modules/couchbase/README.md
+sidebar_label: "couchbase"
+-->
+
+# Couchbase monitoring with Netdata
+
+Couchbase Server is an open source, distributed, JSON document database. It exposes a scale-out, key-value store with managed cache for sub-millisecond data operations, purpose-built indexers for efficient queries and a powerful query engine for executing SQL-like queries.
+
```

Comment:
please add [`## Charts` section](https://github.com/netdata/go.d.plugin/tree/master/modules/dnsmasq#charts)

Reply:
please take a look and let me whether this is enough or not